doc/user_doc/vic_installation/vch_installer_options.md

```diff
@@ -52,10 +52,10 @@ You can designate the same datastore as the image store for multiple virtual con
 
 **NOTES**: 
 - vSphere Integrated Containers supports all alphanumeric characters, hyphens, and underscores in datastore paths and datastore names.
-- In the current builds the `image-datastore` option does not support datastore folders. If you specify a datastore folder in the `image-datastore` option, `vic-machine` does not return an error, but creates all of the necessary folders at the root level of the datastore.
+- In the current builds the `image-store` option does not support datastore folders. If you specify a datastore folder in the `image-store` option, `vic-machine` does not return an error, but creates all of the necessary folders at the root level of the datastore.
```

Comment:
@gigawhitlocks Thank you! I think you're the first person (including me) I've seen update the docs concurrently with the code change. That could be because I'm not reviewing every PR, but still :+1: